calendar.rb

```diff
@@ -0,0 +1,115 @@
+require 'date'
+require_relative 'event'
+require_relative 'home'
+
+
+class Calendar
+  attr_accessor :events
+  def initialize
+    @events = Hash.new { |h, k| h[k] = [] }
+    load_data
+  end
+  
+
+  def load_data
+    file_data = File.open("events.txt");
+    File.foreach("events.txt") do |line|
+      date_from_file, details_from_file = line.split(',')
+      event_data = Event.new(Date.parse(date_from_file),details_from_file.chomp)
+      @events[event_data.date.month].push(event_data)
+    end
+  end
+
+
+  #this method returns the matched events line numbers in an array
+  def get_line_number(file, date)
+    return "incorrect date entered" unless date
+    count = 0
```

Comment:
```suggestion
    return "incorrect date entered" unless date
    
    count = 0
```

Please install rubocop in sublime.